app/views/shared/forms/dependants/_assets_value.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= form_with(
+      model: @form,
+      url: form_path,
+      method: :patch,
+      local: true
+    ) do |form| %>
+
+  <%= govuk_form_group show_error_if: @form.errors.present? do %>
+    <%= govuk_fieldset_header page_title %>
+
+    <div class="govuk-!-padding-bottom-6"></div>
```

Comment:
The `govuk_fieldset_header` helper method has a `padding_below` option. So rather than adding a spacer div, use:
```
<%= govuk_fieldset_header page_title, padding_below: 6 %>